### ISSUE TITLE
Looking for help / maintainers

### ISSUE THREAD

**Issue by boyney123** [boyney123/garie#24]
Hey,

Just wondering if anybody fancied helping out with some of the issues and maintaining the project and plugins?

cc: @zotya as I know you do a lot of work with Garie?

Thanks

**Comment by brennebeck** [comment]
I’ve been following the project. What do you need?

**Comment by boyney123** [comment]
Hey @brennebeck 

Thanks for reaching out.

Looking for help looking at issues, pull requests, documentation, and releases really. 

Interested to hear from people that have the time and some energy to get together and move things forward too with new ideas and features.

**Comment by kafkiano** [comment]
Hey @boyney123 
I want definitely maintain this project further!

I have running a production version of your garie for testing some sites. It works ok but I needed to add some additional bash script for cleaning exited browser time containers and some other stuff. I've added some information about that in run issue here.

I'm not really familiar with developing docker packages but I'm good in javascript. what would be the best way to run your modules locally ?

In conclusion I would really like to talk about garie with you.

**Comment by danipolo** [comment]
I'm interested too.

**Comment by nkeilar** [comment]
@danipolo @felixlberg @boyney123 I'm craving such a solution, but I don't see many updates to the repo lately. Are you still using it - do you recommend this or something else at this point? Great initiative @boyney123 - thanks in advance

**Comment by boyney123** [comment]
Hey @nkeilar ,

Thanks for reaching out, yeah I no longer maintain this project hence the lack of updates and stuff.

I think some people are still using the project but it's hard to tell! If you run into any issues I can try and help but it's been a few years since I look at it.

Here is a great alternative: https://www.sitespeed.io/ (which is open source)

Hope that helps, good luck!
